packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -96,6 +107,8 @@ export class SidebarComponent {
 			await this._click( selector );
 		}
 
+		if ( skipWaitForContent ) return;
```

Comment:
Had to add this as the `waitForSelector` was timing-out in the widgets page.

Reply:
Interesting. I think that perhaps it may make sense to consider removing Line 113 to simplify the method signature at some point. I think we can merge this as-is for now, since removing Line 113 will be a change that will impact all other specs and we don't need to do that now.